sample_consensus/include/pcl/sample_consensus/impl/sac_model_plane.hpp

```diff
@@ -117,30 +117,30 @@ template <typename PointT> inline __m256 pcl::SampleConsensusModelPlane<PointT>:
   return _mm256_andnot_ps (abs_help,
-        _mm256_add_ps (_mm256_add_ps (_mm256_mul_ps (a_vec, _mm256_set_ps (input_->points[(*indices_)[i  ]].x,
-                                                                           input_->points[(*indices_)[i+1]].x,
-                                                                           input_->points[(*indices_)[i+2]].x,
-                                                                           input_->points[(*indices_)[i+3]].x,
-                                                                           input_->points[(*indices_)[i+4]].x,
-                                                                           input_->points[(*indices_)[i+5]].x,
-                                                                           input_->points[(*indices_)[i+6]].x,
-                                                                           input_->points[(*indices_)[i+7]].x)),
-                                      _mm256_mul_ps (b_vec, _mm256_set_ps (input_->points[(*indices_)[i  ]].y,
-                                                                           input_->points[(*indices_)[i+1]].y,
-                                                                           input_->points[(*indices_)[i+2]].y,
-                                                                           input_->points[(*indices_)[i+3]].y,
-                                                                           input_->points[(*indices_)[i+4]].y,
-                                                                           input_->points[(*indices_)[i+5]].y,
-                                                                           input_->points[(*indices_)[i+6]].y,
-                                                                           input_->points[(*indices_)[i+7]].y))),
-                       _mm256_add_ps (_mm256_mul_ps (c_vec, _mm256_set_ps (input_->points[(*indices_)[i  ]].z,
-                                                                           input_->points[(*indices_)[i+1]].z,
-                                                                           input_->points[(*indices_)[i+2]].z,
-                                                                           input_->points[(*indices_)[i+3]].z,
-                                                                           input_->points[(*indices_)[i+4]].z,
-                                                                           input_->points[(*indices_)[i+5]].z,
-                                                                           input_->points[(*indices_)[i+6]].z,
-                                                                           input_->points[(*indices_)[i+7]].z)),
+        _mm256_add_ps (_mm256_add_ps (_mm256_mul_ps (a_vec, _mm256_set_ps ((*input_)[(*indices_)[i  ]].x,
+                                                                           (*input_)[(*indices_)[i+1]].x,
```

Comment:
Can we make a macro and undef it after these lines? Looks to be very easy place to make a error-by-copy in the future.

Reply:
A macro for what exactly? For `(*input_)[(*indices_)[INDEX]]`?

Reply:
Actually, `(*this)[i]` should also work. See pcl_base.h L145

Reply:
But `SampleConsensusModelPlane`does not inherit from `PCLBase` right now. It might make sense to let `SampleConsensusModel` inherit from `PCLBase`.

Reply:
Or implement it if inheritance isn't needed (or complicates matters)